department-of-veterans-affairs/caseflow
# Hearing Prep Header Updates

Resolves #8742 

### Description
Added hearing day attributes to the hearing prep header area.

### Acceptance Criteria

### Testing Plan
1. Go to hearing prep
2. Confirm coordinator, room and notes attributes are present for the hearing day
